pkgs/top-level/aliases.nix

```diff
@@ -393,9 +397,12 @@ mapAliases ({
   gupnp_igd = gupnp-igd; # added 2018-02-25
   gupnptools = gupnp-tools;  # added 2015-12-19
   gutenberg = zola;  # added 2018-11-17
+  gwtdragdrop = throw "gwtdragdrop was removed: abandoned by upstream"; # added 2022-02-06
+  gwtwidgets = throw "gwtwidgets was removed: unmaintained"; # added 2022-02-06
   hawkthorne = throw "hawkthorne has been removed because it depended on a broken version of love"; # added 2022-01-15
   hal-flash = throw "hal-flash has been removed as Adobe Flash Player is now deprecated."; # added 2021-02-07
   heimdalFull = heimdal; # added 2018-05-01
+  heme = throw "heme has been removed: upstream is gone"; # added 2022-02-06
```

Comment:
```suggestion
  gwtdragdrop = throw "gwtdragdrop has been removed because it is abandoned by upstream"; # added 2022-02-06
  gwtwidgets = throw "gwtwidgets has been removed because it is unmaintained"; # added 2022-02-06
  hawkthorne = throw "hawkthorne has been removed because it depended on a broken version of love"; # added 2022-01-15
  hal-flash = throw "hal-flash has been removed as Adobe Flash Player is now deprecated."; # added 2021-02-07
  heimdalFull = heimdal; # added 2018-05-01
  heme = throw "heme has been removed because upstream is gone"; # added 2022-02-06
```